### PR TITLE
Add Cat9554 documet

### PR DESCRIPTION
**Pull request by SchumyHao**
## Description:

Add document for PR: https://github.com/esphome/esphome/pull/908

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>
/esphome/esphome/#908

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Comment by SchumyHao** [comment]
Hi @OttoWinter Can you tell me how to generate svg logo for CAT9554?

There is no pretty image for this IC, I need use svg just show CAT9554

**Comment by SchumyHao** [comment]
Hi @OttoWinter 
Any updata about my request? Waiting for your help :)

**Comment by SchumyHao** [comment]
Hi @OttoWinter
Any updata about my request? Waiting for your help :)

File: components/cat9554.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          inverted: False
+
+Configuration variables:
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+- **id** (**Required**, :ref:`config-id`): The id to use for this CAT9554 component.
+- **address** (*Optional*, int): The I²C address of the driver.
+  Defaults to ``0x20``.
+- **irq_pin** (*Optional*, :ref:`config-pin`): CAT9554 support IRQ mode. If any input pin is changed, IRQ pin will generate falling signal.
+  If irq_pin is not configed, ESPhome will polling input pin status.
```
Comment (glmnet) [inline components/cat9554.rst:47]:
```suggestion
- **irq_pin** (*Optional*, :ref:`config-pin`): CAT9554 support IRQ mode. ESPHome will use IRQ signal to know if polling the I²C is needed. If ``irq_pin`` is not configed, ESPhome will poll every cycle.
```

**Comment by glmnet** [comment]
> Hi @OttoWinter Can you tell me how to generate svg logo for CAT9554?
> 
> There is no pretty image for this IC, I need use svg just show CAT9554

I believe Otto uses InkScape for that

**Pull request closed without merging**